### PR TITLE
Build docker image with Jib

### PR DESCRIPTION
**Pull request by Gseungmin**
### Overall Pull Request
* Change the method of building an image from Dockerfile to Jib.
* Modify the workflow from building an image with Docker and then pushing it to building and pushing with Jib.
* Modify the Docker Compose file.

#### How to run Jib locally:
```
./gradlew jibDockerBuild --image=fosslight/fosslight
```

#### AS-IS: Docker build time
DockerBuild

#### TO-BE: Jib build time
jibDockerBuild

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by jongwooo** [comment]
@Gseungmin Consider changing the title to 'Build docker image with Jib'.

File: docker-compose.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ services:
     expose:
       - "3306"
   web:
-    build: .
+    image: fosslight/fosslight
```
Comment (soimkim) [inline docker-compose.yml:22]: @Gseungmin , 이 수정의 경우, 현재 소스 기반으로 build하는 것이 아닌 dockerhub에 배포된 이미지를 가져오는 것 아닌가요?
docker-compose.yml은 개발시 활용되는 case가 많아 현재 소스 기반으로 빌드 및 실행이 더 좋아보입니다.

**Review by soimkim — CHANGES_REQUESTED** [review]
커맨트로 문의드린 사항에 대해 답변 부탁드립니다.

File: docker-compose.yml (hunk 1 above)
Comment (Gseungmin) [reply docker-compose.yml]: I changed It!

**Pull request merged**